spec/integration/support/serve_examples.rb

```diff
@@ -62,7 +65,28 @@
         "curl http://localhost:5000/descriptions/opensuse131/files/config_files/etc/crontab"
       )
       expect(curl_command).to succeed.with_stderr.and have_stdout(expected_content)
+      # Kill the webserver again
+      @machinery.run_command("pkill -f '#{cmd}'")
+    end
+
+    it "makes the system description HTML available at the config-file port" do
+      @machinery.inject_directory(
+        system_description_dir,
+        machinery_config[:machinery_dir],
+        owner: machinery_config[:owner],
+        group: machinery_config[:group]
+      )
+
+      with_env("MACHINERY_CONFIG_FILE" => "/tmp/config") do
```

Comment:
this file should be cleaned up later on and it is better to use a more uncommon file